client/src/views/Find/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { getLists, setMenuTab } from '../../redux/actions';
+import { Find as FindComponent } from '../../components/Find';
+import './index.css';
+
+class FindClass extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedList: null };
+    this.props.setMenuTab('Find Near Me');
+  }
+
+  componentDidMount() {
+    this.props.getLists();
+  }
+
+  handleSetState = key => e => {
```

Comment:
I guess this is why you use closure on the child component. Don't do this. Separate the handler on each state that you want to work instead.

Reply:
Ah ok, for better readability and simpler code right?

I think I originally wanted to reuse a setState function since they were all doing the same thing, just different key/values

Reply:
If you come back to this app in 5 months, would you know what this thing does in a glance? Most likely not, so prefer readability than DRY.